scripts/getCurrencies.mjs

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
I thought about putting this in `@dinero.js/currencies` but then it would need to be built and need all the overhead of a package when its just a script to get the JSON file. Though if there is a way to import a json file directly from a package, I would be open to that and remove this file.